packages/backend-deployer/src/cdk_error_mapper.ts

```diff
@@ -221,6 +221,16 @@ export class CdkErrorMapper {
       errorName: 'AccessDeniedError',
       classification: 'ERROR',
     },
+    {
+      errorRegex:
+        /User:(.*) is not authorized to perform:(?<action>.*) on resource:(?<resource>.*)/,
```

Comment:
Should we remove the mapping above this as we are essentially capturing the same thing?

Reply:
The one above is capturing a better `action` than the one added here and so I thought about keeping both in this order.